tracks/javascript/exercises/secret-handshake/mentoring.md

```diff
@@ -0,0 +1,63 @@
+### Reasonable solutions
+
+There are two main ways to solve this:
+- using bitwise operations
+- converting to a bitstring and testing for `'1'`
+
+### Bitwise solution
+
+```javascript
+const OPERATIONS = [
+  'wink',               // idx: 0 => 1 << 0 => 0b00000001 (1)
+  'double blink',       // idx: 1 => 1 << 1 => 0b00000010 (2)
+  'close your eyes',    // idx: 2 => 1 << 2 => 0b00000100 (4)
+  'jump'                // idx: 3 => 1 << 3 => 0b00001000 (8)
+]
+
+export function secretHandshake(sekret) {
+  if (!(typeof sekret === 'number')) {
+    throw new Error('Handshake must be a number')
+  }
+
+  const shake = OPERATIONS.filter((_, i) => sekret & (1 << i))
+  return sekret & 16 ? shake.reverse() : shake
+}
+```
+
+Note that the `secret & 16` can also be written as `1 << 4` or `0b10000` or
```

Comment:
`sekret` or `secret` ?

Reply:
Ah good catch. Thank you! It was sekret on purpose indeed